Content.Client/GameObjects/Components/Storage/BagOpenCloseVisualizer.cs

```diff
@@ -0,0 +1,63 @@
+﻿#nullable enable
+
+using Content.Shared.GameObjects.Components;
+using JetBrains.Annotations;
+using Robust.Client.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Utility;
+using YamlDotNet.RepresentationModel;
+
+namespace Content.Client.GameObjects.Components.Storage
+{
+    [UsedImplicitly]
+    public class BagOpenCloseVisualizer : AppearanceVisualizer
+    {
+        private const string OpenIcon = "openIcon";
+        private string? _openIcon;
+
+        public override void LoadData(YamlMappingNode node)
+        {
+            base.LoadData(node);
+
+            if (node.TryGetNode<YamlScalarNode>(OpenIcon, out var openIconNode))
+            {
+                _openIcon = openIconNode.Value;
```

Comment:
Is there any way to destroy this visualizer if this is not the case? If not just make an issue as it could be useful.

Reply:
Ok. I don't remember destroying visualizers, but it's fair.

Reply:
Also should've mentioned just make it log a warning instead in an else statement.